src/sonic-config-engine/minigraph.py

```diff
@@ -261,17 +261,15 @@ def parse_cpg(cpg, hname):
                 end_router = session.find(str(QName(ns, "EndRouter"))).text
                 end_peer = session.find(str(QName(ns, "EndPeer"))).text
                 if end_router == hname:
-                    bgp_sessions.append({
+                    bgp_sessions[start_peer] = {
                         'name': start_router,
-                        'addr': start_peer,
                         'peer_addr': end_peer
-                    })
```

Comment:
As the table name is "BGP_NEIGHBOR", I think it is pretty straight forward that "name" here is the name of bgp neighbor. "peer_addr" does sound a bit confusing, I'll suggest we can change it into "local_addr". What's your suggestion?